azext_iot/central/_help.py

```diff
@@ -1752,3 +1754,257 @@ def _load_central_edge_help():
+        "iot central enrollment-group list"
+    ] = """
+    type: command
+    short-summary: Get the list of enrollment groups in an application
+    examples:
+      - name: Get the list of enrollment groups in an application
+        text: >
+          az iot central enrollment-group list
+          --app-id {appid}
+    """
+
+    helps[
+        "iot central enrollment-group show"
+    ] = """
+    type: command
+    short-summary: Get details about an enrollment group by ID
+    examples:
+      - name: Get details about an enrollment group by ID
+        text: >
+          az iot central enrollment-group show
+          --app-id {appid}
+          --id {enrollmentGroupId}
+    """
+
+    helps[
+        "iot central enrollment-group delete"
+    ] = """
+    type: command
+    short-summary: Delete an enrollment group by ID
+    examples:
+      - name: Delete an enrollment group by ID
+        text: >
+          az iot central enrollment-group delete
+          --app-id {appid}
+          --id {enrollmentGroupId}
+    """
+
+    helps[
+        "iot central enrollment-group create"
+    ] = """
+    type: command
+    short-summary: Create an enrollment group
+    examples:
+      - name: Create an enrollment group
+        text: >
+          az iot central enrollment-group create
+          --app-id {appid}
+          --id {enrollmentGroupId}
+          --at {attestation}
+          --display-name {displayName}
+          --type {type}
+
+      - name: Create an enrollment group with x509 primary certification
+        text: >
+          az iot central enrollment-group create
+          --app-id {appid}
+          --id {enrollmentGroupId}
+          --at 'x509'
+          --display-name {displayName}
+          --type {type}
+          --cp {primayCertPath}
+
+      - name: Create an enrollment group with given symmetric key certification
+        text: >
+          az iot central enrollment-group create
+          --app-id {appid}
+          --id {enrollmentGroupId}
+          --at 'symmetricKey'
+          --display-name {displayName}
+          --type {type}
+          --pk {primaryKey}
+          --sk {secondaryKey}
+    """
+
+    helps[
+        "iot central enrollment-group update"
+    ] = """
+    type: command
+    short-summary: Update an enrollment group
+    examples:
+      - name: Update an enrollment group
+        text: >
+          az iot central enrollment-group update
+          --app-id {appid}
+          --id {enrollmentGroupId}
+          --display-name {displayName}
+          --type {type}
+          --ps {enabled}
+
+      - name: Remove x509 primary certificate from an enrollment group
+        text: >
+          az iot central enrollment-group update
+          --app-id {appid}
+          --id {enrollmentGroupId}
+          --remove-x509 'true'
+          --entry 'primary'
+    """
+
+    helps[
+        "iot central enrollment-group verify-certificate"
+    ] = """
+    type: command
+    long-summary: |
```

Comment:
Can we add a short-summary for `verify-certificate`?  It's currently blank in the `enrollment-group` help:
![image](https://user-images.githubusercontent.com/13545962/192354172-3641abe1-ad20-4488-a40e-3e5355318ae7.png)

Reply:
Sure we can.

Reply:
Added.